urbs/model.py

```diff
@@ -485,9 +490,13 @@ def create_model(data, dt=1, timesteps=None, dual=False):
         rule=res_storage_capacity_rule,
         doc='storage.cap-lo-c <= storage capacity <= storage.cap-up-c')
     m.res_initial_and_final_storage_state = pyomo.Constraint(
-        m.t, m.sto_tuples,
+        m.t, m.sto_init_bound_tuples,
         rule=res_initial_and_final_storage_state_rule,
         doc='storage content initial == and final >= storage.init * capacity')
+    m.res_initial_and_final_storage_state_var = pyomo.Constraint(
+        m.t, m.sto_tuples - m.sto_init_bound_tuples,
+        rule=res_initial_and_final_storage_state_var_rule,
+        doc='storage content initial <= final, both variable')
```

Comment:
Probably not, but still I wonder why. We also have the definition that the initial has to be the same as the final state, and this is with equality. Hence, I am confused, why to change it to this.

Reply:
In fact, I took the inequality from the current version of urbs where it is also written as an inequality. The reason I guess is that a) the optimization gains quite some flexibility to avoid potential steep cost increases while b) the main reason for this constraint, namely the avoiding of storage emptying benefits, are still avoided as Soner pointed out.

Reply:
I think in general taking inequalities when possible is computationally cheaper and a little better from the optimization perspective.